src/app/blogs/[id]/page.tsx

```diff
@@ -26,16 +26,24 @@ const post = {
 // TODO dummy page (Needs properly formatted page that accepts WYSIWYG)
 const BlogPost = () => {
   return (
-    <div className="mx-auto max-w-4xl p-4">
+    <div className="mx-auto min-h-screen max-w-4xl p-4">
       <div className="mb-4">
-        <img src={post.image} alt={post.title} className="rounded-lg" />
+        <Image
```

Comment:
Linter says `Image` is better 🤓 
![Screenshot 2024-07-16 at 10 18 30 AM](https://github.com/user-attachments/assets/424b3795-bb58-4bc0-81a9-ead3b8339185)